src/systems/ec/specs/components/link/docs/usage.md

```diff
@@ -1 +1,53 @@
-Do's and don'ts
+## When to use
+
+When you need to direct users to a different page or to provide access to further information.
+
+## Link types
+* in-line text link
+  * internal links
+  * external links
+* stand-alone link (have no underline)
+  * more link
+  * expandable link
+  * social media link
+
+## When to use this component
+
+### in-line link
+
+When links are inserted within paragraph text blocks
+Style blue, underlined (displayed in white colour if the background colour is blue)
+
+#### internal link
+
+when links point to URLs inside the ec.europa.eu domain
+style 
+underlined link
+
+#### external link
+
+When links point to URLs outside the ec.europa.eu domain
+style
+underlined link with an appended square w/ arrow pointing outward
+
+### stand-alone link
+
+When links are not inserted within paragraph text blocks
+style
+blue, no underline
+
+#### more link
+
+Used on the Listings component
+
+#### expandable link
+
+Used on the Expandable component
+
+#### social media link
+
+Used on the Social media - Follow. and Social media - Share. components
+
+## When not to use 
+
+For actions or important links to other pages -  use buttons instead
```

Comment:
Please do not confuse links with buttons. Links are used for navigation. Buttons are used for actions (e.g. submitting/resetting a form).

- http://www.webaxe.org/proper-use-buttons-links/
- https://medium.com/simple-human/but-sometimes-links-look-like-buttons-and-buttons-look-like-links-9b371c57b3d2

Reply:
https://a11y-101.com/design/button-vs-link

Reply:
I will make sure the relevant person to check it and updated. I just copy paste the content that I have in the wiki. Consider it as a draft.